.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: Feature request
 about: Suggest an idea/feature for this project
+labels: feature
```

Comment:
we don't have a feature label defined: https://github.com/rook/rook/labels.  What do you think about `enhancement`? https://github.com/rook/rook/labels/enhancement

Reply:
What if we rename `enancement` to `feature`?

Reply:
That's fine with me :)

Reply:
As there don't seem to be objections against renaming the label, I went ahead and renamed it.